src/python/friendzone/nwx2ase/__init__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2024 NWChemEx-Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ..friends import is_friend_enabled
+from .nwchem_via_ase import NWChemEnergyViaASE, NWChemGradientViaASE
+
+
+def load_ase_modules(mm):
+    if is_friend_enabled('nwchem'):
```

Comment:
Should this be `is_friend_enabled('ase')`? Or possibly even `is_friend_enabled('ase') and is_friend_enabled('nwchem')`?

Reply:
Yeah I meant for it to be the latter but never added the ASE check. I'll push the fix whenever I also figure out the docs error.